vispy/app/tests/test_context.py

```diff
@@ -68,9 +70,9 @@ def check():
         # Check while c1 is active
         check()
 
-        # pyqt5 does not currently support context sharing
-        if 'pyqt5' in c1.app.backend_name.lower() or 'pyqt6' in c1.app.backend_name.lower():
-            pytest.xfail("Context sharing is not supported in PyQt5 at this time.")
+        # pyqt5 does not currently support context sharing, pyside6 seg faults on app tests
+        if 'pyqt5' in c1.app.backend_name.lower() or 'pyqt6' in c1.app.backend_name.lower() or 'pyside6' in c1.app.backend_name.lower():
+            pytest.xfail("Context sharing is not supported in PyQt5, PyQt6, PySide6 at this time.")
```

Comment:
Pyside2 also, right?

Reply:
I don't know and I can't test, because pyside2 seems borked on arm64.
Pyside6 the context sharing works and the test passes if run individually, but if you run the app suite then you get seg fault.

Reply:
Oh, of course you're right. I'll add the xfail for pyside2 then.